### PR TITLE
Match locale for map with bolt config

### PR DESCRIPTION
**Pull request by ashceo**
Added a parameter to map to match bolt locale instead of client locale.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,6 +27,9 @@
     "entrypoint": "Bolt\\Geolocation\\Extension",
     "screenshots": [
       "screenshots/backend-zoom.gif"
-    ]
+    ],
+    "branch-alias": {
+      "dev-master": "1.x-dev"
+    }
```
Comment (LordSimal) [inline composer.json]: why is this needed?

Comment (ashceo) [reply composer.json]: My bad, for upstream not needed, removed.
